scout/server/blueprints/variant/utils.py

```diff
@@ -148,6 +148,26 @@ def update_transcripts_information(variant_gene, hgnc_gene, variant_obj, genome_
         transcript["change_str"] = transcript_str(transcript, hgnc_symbol)
 
 
+def update_variant_case_panels(store, case_obj, variant_obj):
+    """Populate variant with case gene panels with info on e.g. if a panel was removed on variant_obj.
+    Variant objects panels are only a list of matching panel names.
+
+    Args:
+        store(adapter.MongoAdapter)
+        case_obj(dict)
+        variant_obj(dict)
+    """
+
+    variant_panel_names = variant_obj.get("panels") or []
+    case_panel_objs = [
+        panel
+        for panel in (case_obj.get("panels") or [])
```

Comment:
I would still make it check that the panels are not hidden, since relies of a filtering happening before that could not be in place once we forget about it